tests/test_packages/test_skills_integration/test_tac.py

```diff
@@ -435,6 +439,14 @@ def test_tac(self):
             diff == []
         ), "Difference between created and fetched project for files={}".format(diff)
```

Comment:
This assert in the newly changed test used to fail when I ran it locally (the agents are not the same because here we have the additional `is_https` in the config set to False. In the original package that is not present.)

Reply:
The order matters!